match name change in registry-common

al-niessner opened the pull request:
## 🗒️ Summary
Simple name change to match registry-common.

## ⚙️ Test Data and/or Report
NA

## ♻️ Related Issues
Depends upon NASA-PDS/registry-common#74

tloubrieu-jpl commented:
Hi @al-niessner ,

I tested the `set-archive-status` and `delete-data` subcommands on MCP dev.

The `set-archive-status` works with the `-lidvid` and `-packageId` options.
`delete-data` worked with the `-lidvid` option but I had an error with the `-packageId` option:
 ```
% ./bin/registry-manager delete-data \
    -auth /Users/loubrieu/Documents/pds/registry/es-auth.txt \
    -es file:/Users/loubrieu/Documents/pds/registry/mcp_dev.xml \
    -packageId 97f89d9c-3f15-4142-8ee0-098504e17e2b                            

Elasticsearch URL: file:/Users/loubrieu/Documents/pds/registry/mcp_dev.xml
null

SLF4J: Failed to load class "org.slf4j.impl.StaticLoggerBinder".
SLF4J: Defaulting to no-operation (NOP) logger implementation
SLF4J: See http://www.slf4j.org/codes.html#StaticLoggerBinder for further details.
[ERROR] Request failed: [http_exception] server returned 404
```

Any idea what the issue is ? I believe it previously did not return errors but was missing to delete some of the documents.

al-niessner commented:
Investigating...

When run using local DB (docker opensearch:2.13.0) it works normally:
```
Deleted 47 document(s) from registry index
Deleted 9 document(s) from registry-refs index
```

When run using AWS or local and wrong package id:
```
Deleted 0 document(s) from als_test_registry index
Deleted 0 document(s) from als_test_registry-refs index
```

@tloubrieu-jpl @jordanpadams 
When run using AWS and correct package id get same error and could reproduce. Wasted half a day only to figure out serverless does not support scroll. Back peddling to figure out something else...

tloubrieu-jpl reviewed (approved):
Previously failing test described in comment now works.
